ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -486,6 +486,14 @@ tests:
         export WORKER_COUNT=0
       PLATFORM: vsphere
     workflow: assisted-vsphere
+- as: e2e-metal-assisted-ipxe
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        export IPXE_BOOT=true
```

Comment:
Seems like you should remove the "export"
At least by judging from most of the job definitions

Reply:
Removed it, thank you!